src/main/drivers/sonar_hcsr04.c

```diff
@@ -134,30 +138,33 @@ void hcsr04_init(const sonarHardware_t *initialSonarHardware, sonarRange_t *sona
     NVIC_InitStructure.NVIC_IRQChannelCmd = ENABLE;
     NVIC_Init(&NVIC_InitStructure);
 
-    lastMeasurementAt = millis() - 60; // force 1st measurement in hcsr04_get_distance()
-#else
-    lastMeasurementAt = 0; // to avoid "unused" compiler warning
 #endif
 }
 
-// measurement reading is done asynchronously, using interrupt
+#ifdef UNIT_TEST
+void digitalHi(void *p, uint16_t i) {UNUSED(p);UNUSED(i);}
```

Comment:
Test code is full of repeated stubs ;-)  ... IMO sonar test instrumentation is a bit too intrusive, but I can live with it .. 

But rest of sonar_hcsr04.c code does not use digitalIO when compiled with UNIT_TEST, so I thought that either `#if !defined(UNIT_TEST)` of these stubs were left by mistake